src/api/app/views/webui/request/show.html.erb

```diff
@@ -145,6 +145,14 @@
                 wants the role <b><%= action[:role] %></b> for
                 <%= project_or_package_link project: action[:tprj], package: action[:tpkg] %>
               </b>
+          <% elsif action[:type] == :set_bugowner %>
+              <b>
+                <%= user_with_realname_and_icon @bs_request.creator %>
+                wants to add
```

Comment:
correction: this does not add, this sets(replaces) the bugowner.

Reply:
right, "wants to become" bugowner is the right wording IMHO

Reply:
Nono! This is also wrong for the maintainer. Because the @bs_request.creator is not necessarly the one set.

Reply:
'coolo wants adrian to become bugowner' would be acceptable though :)

And test cases would be great - as it's not the first time this broke

Reply:
right :)

Reply:
@coolo 

> 'coolo wants adrian to become bugowner' would be acceptable though :)

I can change it, there is not much difference through. :thinking: 

> And test cases would be great - as it's not the first time this broke

We are not testing the views, I do not think we need to test this. @openSUSE/open-build-service opinions?

Reply:
about the text: just ignore everything adrian said - just correct the verb :)

Reply:
Well, your lack of testing the views is obvious - not sure it's a good argument :)

Reply:
@coolo 

> Well, your lack of testing the views is obvious - not sure it's a good argument :)

we test the logic, the view doesn't have logic, just render things.

Reply:
Now I'm curious: what's your limit on the number of if/else to call it logic? :stuck_out_tongue_winking_eye: